draft-ietf-quic-tls.md

```diff
@@ -1183,48 +1262,47 @@ SHOULD track redundant packets and treat excessive volumes of any non-productive
 packets as indicative of an attack.
 
 
-## Packet Number Protection Analysis {#pn-encrypt-analysis}
+## Header Protection Analysis {#pn-encrypt-analysis}
 
-Packet number protection relies on the packet protection AEAD being a
-pseudorandom function (PRF), which is not a property that AEAD algorithms
+Header protection relies on the packet protection AEAD being a pseudorandom
+function (PRF), which is not a property that AEAD algorithms
 guarantee. Therefore, no strong assurances about the general security of this
 mechanism can be shown in the general case. The AEAD algorithms described in
 this document are assumed to be PRFs.
 
-The packet number protection algorithms defined in this document take the
-form:
+The header protection algorithms defined in this document take the form:
 
 ~~~
-encrypted_pn = packet_number XOR PRF(pn_key, sample)
+protected_field = field XOR PRF(pn_key, sample)
 ~~~
 
 This construction is secure against chosen plaintext attacks (IND-CPA) {{IMC}}.
 
 Use of the same key and ciphertext sample more than once risks compromising
-packet number protection. Protecting two different packet numbers with the same
-key and ciphertext sample reveals the exclusive OR of those packet numbers.
-Assuming that the AEAD acts as a PRF, if L bits are sampled, the odds of two
-ciphertext samples being identical approach 2^(-L/2), that is, the birthday
-bound. For the algorithms described in this document, that probability is one in
-2^64.
+header protection. Protecting two different headers with the same key and
+ciphertext sample reveals the exclusive OR of the protected fields.  Assuming
+that the AEAD acts as a PRF, if L bits are sampled, the odds of two ciphertext
+samples being identical approach 2^(-L/2), that is, the birthday bound. For the
+algorithms described in this document, that probability is one in 2^64.
 
```

Comment:
Is this probality still true for PN's shorter than 4 bytes? Or even at all given that the mask is at most only 36  or 37 bits effectively?

Reply:
I forget that the sample is a block size input to a encryption step.